code/modules/mob/living/silicon/ai/freelook/eye.dm

```diff
@@ -16,6 +16,7 @@
 	var/relay_speech = FALSE
 	var/use_static = TRUE
 	var/static_visibility_range = 16
+	var/AiDetector = TRUE // Decides if it is shown by AI Detector or not
```

Comment:
```suggestion
	/// Decides if it is shown by AI Detector or not
	var/show_on_multitool = TRUE
```
Autodoc compliance, better var name, as well as case compliance

Reply:
Fixed the proposed change, it was used more than just those two places, now replaced all the places.